packages/web-components/src/components/cta/video-cta-composite.ts

```diff
@@ -78,7 +78,7 @@ class DDSVideoCTAComposite extends ModalRenderMixin(HostListenerMixin(LitElement
   private async _handleRequestVideoData(event: CustomEvent) {
     const { href } = event.detail;
     (event.target as VideoCTAMixinImpl).videoThumbnailUrl = VideoPlayerAPI.getThumbnailUrl({
```

Comment:
Hey @guilhermelMoraes, I'm thinking we probably want to switch out `VideoPlayerAPI` with `KalturaPlayerAPI` nothing will break when we eventually remove `VideoPlayerAPI` from the codebase.

Reply:
Makes sense. Will do that, @annawen1